lib/token.js

```diff
@@ -302,6 +302,18 @@ function handleOAuthResponse(sdk, oauthParams, res) {
     }
   }
 
+  if (res['code']) {
```

Comment:
Random note: I noticed on line 298 we're using Array.isArray, but in the last commit we added Util.isArray - I don't mind which one we pick, but let's be consistent.